services/redis.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: sd-redis
+  labels:
+    name: redis
```

Comment:
updated all names to prefix with `sd-redis`, so that our names/labels always have `sd` prefix.